packages/core/src/types/IViewport.ts

```diff
@@ -11,14 +11,24 @@ import type Viewport from '../RenderingEngine/Viewport';
  * not currently in view, such as for a different slice, or containing a given
  * set of points.
  */
-export interface ViewReferenceSpecifier {
+export type ViewReferenceSpecifier = {
   /**
    * The slice index within the current viewport camera to get a reference for.
-   * Note that slice indexes are dependent on the particular view being shown
-   * and cannot be shared across different view types such as stacks and
-   * volumes, or two viewports showing different orientations or slab thicknesses.
+   * Note that slice indexes are dependent on the data in a viewport, and will
+   * be used to generate a view reference or referenced image, but are not the
+   * reference itself.
    */
-  sliceIndex?: number | [number, number];
+  sliceIndex?: number;
+  /**
+   * The end index - this requires sliceIndex to be specified.
+   * This will result in a view reference containing a range, specified
+   * by the starting and ending images or by a thickness for a volume.
+   */
+  sliceRangeEnd?: number;
+
```

Comment:
How about `rangeEndSliceIndex` and below instead of `endRangeImageURI` we replace with `rangeEndImageURI` (I assume there should be a `rangeEndImageId` too?)